### PR TITLE
Milestone 1: Setup project structure of Vet Clinic App

### PR DESCRIPTION
**Pull request by Felix45**
- Setup the project structure according to the provided [template](https://github.com/microverseinc/curriculum-databases-projects-template)
- Created the `vet_clinic` database
- Created the `animals` table in the `vet_clinic` database
- Populated the `animals` table in the `vet_clinic` database with data
- Queried the `animals` table in `vet_clinic` database
- Added the application screenshot to the project

## Queries screenshot
Queries ScreenShot

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name, neutered, escape_attempts FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
```
Comment (mikethreels) [inline queries.sql]: - [ ] in this case you should only return the Name of the animal

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name, date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```
Comment (mikethreels) [inline queries.sql]: - [ ] in this case you should only return the date_of_birth

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Felix, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Felix45) [reply queries.sql]: Hi, @mikethreels
I've implemented the required changes.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Felix45 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**